### DISCUSSION TITLE
Restart Office hours on Slack!

### DISCUSSION THREAD

**Discussion by OrlinVasilev** [vmware-tanzu/velero#5530] [Community meetings]
Hi All,
 in todays Community meeting(November 1, 20222) we discussed that we may restart Office Hours async in slack #velero-users in specific time slot which can include US and Beijing folks!
![image](https://user-images.githubusercontent.com/7236111/199261704-8109d879-6112-45a1-9c48-73af3635d7af.png)

Please share your thoughts if that will work for you and if you are fine with the time slot to be monitoring slack for questions and if you can help!

To start like every second Monday and to monitor if it needs more time or different time or even zoom!

Thank you all!
Please share thought bellow :)
Orlix

**Comment by weshayutin** [comment]
Thanks @OrlinVasilev +1 from me

**Comment by kaovilai** [comment]
I think having everyone monitor the same time slot while they are in different timezone may be overkill. Too few users, too many people to answer questions.

How about 2 hours per timezone's working hours for 3-4 timezones  so we end up with 6-8 hours total of async office hours. Anything left unanswered (but acknowledged by timezone owners) can be addressed during the next community meetings.

This approach encourages maintainer/contributor to learn more about every aspect of velero and not get siloed into their specialties that only they can answer.

As for having synchronous time to chat about a problem, that's what community meetings are for right?
